src/__tests__/index.spec.ts

```diff
@@ -48,6 +48,34 @@ const B = /*#__PURE__*/String("b");`,
   title: "Annotated #__PURE__ for fn(fn())",
   src: `export const A = fn(fn())`,
   dest: `export const A = /*#__PURE__*/fn( /*#__PURE__*/fn());`,
+}, {
+  title: "Annotate only top level calls",
+  src: `export const A = () => { var B = call(); };`,
+  dest: `export const A = () => {
+  var B = call();
+};`,
+}, {
+  title: "Treat calls in top level IIFEs as top level calls",
```

Comment:
Without the top level checker, this case can work too.

Reply:
sure, because without it the plugin just annotates every call, no matter of the position in the file

Reply:
This is the key point. 
We need annotates every call, no matter of the position in the file.
The only thing we need to care, that is the call result of a function is used.